src/main/java/org/blockchain_innovation/factom/identiy/did/LowLevelIdentityClient.java

```diff
@@ -98,13 +96,7 @@ public EntryApi getEntryApi() {
                                 .setChainId(entryResponse.getChainId())
                                 .setExternalIds(entryResponse.getExtIds())
                                 .setContent(entryResponse.getContent()));
-
-
-                try {
-                    entries.add(ENTRY_FACTORY.from(entry, blockInfo, validate));
-                } catch (RuleException re) {
-
-                }
+                entries.add(ENTRY_FACTORY.from(entry, blockInfo, validate));
```

Comment:
This means the code will break on an invalid entry instead of skipping the entry. The previous code was right, except missing the logging part of it

Reply:
Updated.